dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/worker/task/TaskManager.java

```diff
@@ -69,8 +74,9 @@ public static AbstractTask newTask(TaskExecutionContext taskExecutionContext, Lo
             case SQOOP:
                 return new SqoopTask(taskExecutionContext, logger);
             default:
-                logger.error("unsupport task type: {}", taskExecutionContext.getTaskType());
-                throw new IllegalArgumentException("not support task type");
+                String msg = String.format("not support task type: %s", taskExecutionContext.getTaskType());
```

Comment:
it will be better to use log pattern，not string format.

Reply:
done